lnwallet/fee_estimator.go

```diff
@@ -15,6 +21,25 @@ const (
 	// FeePerKwFloor is the lowest fee rate in sat/kw that we should use for
 	// determining transaction fees.
 	FeePerKwFloor SatPerKWeight = 253
+
+	// MaxBlockTarget is the highest number of blocks confirmations that
+	// a WebAPIFeeEstimator will cache fees for. This number is chosen
+	// because it's the highest number of confs bitcoind will return a fee
+	// estimate for.
+	MaxBlockTarget uint32 = 1009
```

Comment:
looks like these constants don't have to be exported?

Reply:
fixed :sweat_smile: